src/pytom_tm/angles.py

```diff
@@ -88,6 +88,8 @@ def get_angle_list(angle: Union[pathlib.Path, float], sort_angles: bool = True,
           maximum difference (in degrees) for the angle list
     sort_angles: bool, default True
         sort the list, using python default angle_list.sort(), sorts first on Z1, then X, then Z2
+    symmetry: int, default 1
+        the returned list will only have Z2 angles [0, (2*pi/symmetry))
```

Comment:
Could you add here that the symmetry is not applied for files? A similar message would also be handy to have at the command line argument for rotational symmetry.

Reply:
But it is applied to files?
files are read in in line `118` and then filtered in line `122`?
(this was also the case in the old implementation, hence why I left it that way

Reply:
Woops, read it wrong!